### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by Jorindel53** [caligraf/ChangeQuote#11]
Hello, 

I installed ChangeQuote on Thunderbird 68.4.1 (32-bit) and it does not change the reply header. 

The header ist still the original one:
On 22.01.2020 12:10, XYZ wrote:

Other Addons I have:
Allow HTML Temp
Lightning
LookOut (fix version)
Mail Merge

Do you see any conflicts? Are there other reasons which could stop ChangeQuote to work?

**Comment by caligraf** [comment]
Hello,
There was a bug that I have fixed in version 1.3 : if the message is opened in a new window the reply header is incorrect or the original one.
Which version do you have ?

**Comment by caligraf** [comment]
Do you still have the issue with latest version 1.5.1 [https://github.com/caligraf/ChangeQuote/releases/tag/1.5.1](https://github.com/caligraf/ChangeQuote/releases/tag/1.5.1) ?

**Comment by the-solipsist** [comment]
I'm using ChangeQuote version 1.5.1 with Thunderbird 78.5.0 on Linux (64 bit), and while it is marked as being compatible, it isn't working.

What can I do to debug the problem?

**Comment by lihatt** [comment]
Same as what the-solipsist said. Tb 78.5.0 32-bit running on Windows 64-bit, and the extension is having no effect on the reply headers.

**Comment by caligraf** [comment]
@the-solipsist Can you check if you have some error messages in the error console ?
The error console is available from the menu Tools > Developper Tools > Error Console or with Ctrl + Shift + J

**Comment by lihatt** [comment]
Okay, strange... in the process of trying to determine whether the warning  _window.controllers/Controllers is deprecated. Do not use it for UA detection._ that appears in the console when I reply to a message was possibly connected to changeQuote, I disabled the extension (figuring that if the warning still appeared, it could not be changeQuote causing it) - and now it works as expected after re-enabling it.

**Comment by the-solipsist** [comment]
Same thing as @lihatt, but even after re-enabling it didn't work a few times, and then it started working partially.  It is now working for 3 of my 4 accounts, but inexplicably the date format for the fourth doesn't change regardless of what custom date format I use.

Here's the console log I obtained from the changeQuote "debugging extensions" page: https://gist.github.com/the-solipsist/75435d8ceb5f3fc2768fc29d2554abda

**Comment by lihatt** [comment]
I don't have a lot of testing samples yet but what seems to be happening for me is that it does not work on Tb initial startup, and then if I disable and re-enable changeQuote it works. So far it appears to be affecting all eight accounts consistently (including a Gmail account and an MS Exchange account alongside more "vanilla" IMAP accounts, if that makes any difference; but I'm not sure why it would).

**Comment by caligraf** [comment]
Do you have other extensions installed ?

**Comment by lihatt** [comment]
Yes., four: Allow HTML Temp, Compact Headers, Manually sort folders, and Message Header Toolbar Customize. Disabling them all made changeQuote function normally - that is, I didn't have to fiddle with disabling and re-enabling it to make it work. So I turned the others back on one by one and the one that conflicted was Allow HTML Temp. Enabling that one will produce the previously observed behavior in changeQuote, that it doesn't actually do its thing unless I disable and then re-enable it.

I find this surprising, because I wouldn't have thought these two extensions tinker with the same area of Thunderbird, but maybe there's something going on I'm not aware of. Allow HTML Temp is this: https://addons.thunderbird.net/en-US/thunderbird/addon/allow-html-temp/

**Comment by the-solipsist** [comment]
I can confirm that disabling "Allow HTML Temp" ([source code](https://gitlab.com/ThunderbirdMailDE/allow-html-temp)) fixes one problem: that of having to disable and re-enable it.  Thanks for figuring that out, @lihatt!  

However, I'm still facing the problem where ChangeQuote works for 2 of my mail accounts, but not the 3rd.

I checked the mail headers to see if the `Date:` headers are different, but that wasn't the case.  I sent a test e-mail from a Gmail address to all 3 e-mails.  

I have ChangeQuote use "Customized header reply", with the format: `<%%7> [%%5]:`, and "Use long or customized format in date field" selected, along with the "Customized format (sender)" set to: `%Y-%m-%d | %H:%i:%s %z`.


The `Date:` header in that test e-mail in all three account reads:
`Date: Sun, 13 Dec 2020 01:15:10 +0530`

But when I press Ctrl+R, the reply header format is different for one of the mail accounts:
Account 1: `<name@example.com> [Sun Dec 13 2020 01:15:10 GMT+0530 (India Standard Time)]:`
Account 2: `<name@example.com> [2020-12-13 | 01:15:10 +0530]:`
Account 3: `<name@example.com> [2020-12-13 | 01:15:10 +0530]:`

I can't seem to figure out what about Account 1 is causing this problem.  I checked all the settings in "Account Settings", but there seem to be no significant differences between the three accounts.

**Comment by the-solipsist** [comment]
I think the issue I'm facing now is different from the interference with "Allow HTML Temp", and so I'm opening a new issue instead of crowding this one.

**Comment by lihatt** [comment]
Yeah, I can't reproduce the behavior you are seeing. It works as expected on all seven of my accounts. (Yes, I had previously said eight - one has since been deleted.)

**Comment by arandorion** [comment]
I'm running Change Quote 1.5.3 on Thunderbird 91.10.0 (macOS), and I am experiencing the same conflict with "Allow HTML Temp". I either have to disable "Allow HTML Temp", or I have to disable then re-enable "Change Quote" in order for it to work properly.

**Comment by caligraf** [comment]
The add-on ChangeQuote has been rewritten for Thunderbird 115, could you test with the last version 1.7.9 ?
https://github.com/caligraf/ChangeQuote/releases/download/1.7.9/ChangeQuote.xpi
